report/01_Introduction.md

```diff
@@ -1,10 +1,13 @@
-# Report
+# Devoops Project Report
 
-This report details our project done February to May in 2020 during the DevOps project.
+This document is the final report of Group D(evOops) for course "DevOps, Software Evolution and Software Maintenance" at IT University of Copenhagen. The report covers a project done in the period of February to May of 2020 by the following members:
+- Hjalte Sorgenfrei Mac Dalland, hjda@itu.dk
+- Jakob Isrealsen, jais@itu.dk
+- Jonas Lindenskov Nielsen, joln@itu.dk
+- Tobias Løfgren, tobl@itu.dk
+- Simon Green Kristensen, sigk@itu.dk
 
-The team consisted of:
-- Hjalte Sorgenfrei Mac Dalland 
-- Jakob Isrealsen
-- Jonas Lindenskov Nielsen
-- Tobias Løfgren
-- Simon Green Kristensen
+The intent of the course was to learn how to maintain and evolve a legacy system. 
+For this purpose a legacy twitter clone, implemented in Flask with Python 2, was given to the group and the task with refactoring the application to be more modern. 
```

Comment:
"More modern" is both vague and unacademic. Maybe "to be able to run on modern systems, using a more modern language" (although I hate using modern twice in a sentence). Maybe also add something about following DevOps practices, like CI/CD and stuff like monitoring and logging